File: src/main/java/gargoyle/ct/convert/impl/ShortConverter.java
package gargoyle.ct.convert.impl;

import gargoyle.ct.convert.Converter;
import org.jetbrains.annotations.NotNull;

public class ShortConverter implements Converter<Short> {
    @Override
    public String format(Short data) {
        return String.valueOf(data);
    }

    @NotNull
    @Override
    public Short parse(@NotNull String data) {
        return Short.valueOf(data);
    }
}

File: src/main/kotlin/gargoyle/ct/convert/impl/ShortConverter.kt
package gargoyle.ct.convert.impl

import gargoyle.ct.convert.Converter

class ShortConverter : Converter<Short> {
    override fun format(data: Short): String {
        return data.toString()
    }

    override fun parse(data: String): Short {
        return data.toShort()
    }
}
